ci/README.md

```diff
@@ -0,0 +1,98 @@
+# VIP Go CI Scripts
+
+This directory is not deployed to your VIP site.
+
+## `deploy.sh` and `deploy-exclude.txt`
+
+Scripts designed to facilitate building your Javascript and CSS
+on either [Travis CI](https://travis-ci.com) or [Circle CI](https://circleci.com/).
+
+See our documentation here: (NEEDS LINK, Ed)
+
+### [Circle CI](https://circleci.com/)
+
+Sample Circle CI config, put into `.circleci/config.yml`:
+
+``` yml
+version: 2
+jobs:
+  build:
+    docker:
+      # Pick a base image which matches the version of Node you need for
+      # building from https://hub.docker.com/r/circleci/node/tags/
+      #
+      # Note: If using a different container, make sure it contains at least
+      # git 2.6.0. (Use -stretch for circleci/node containers.)
+      - image: circleci/node:6.11-stretch
+
+    branches:
+      # DEPLOY: Don't build from a branch with the `-built` suffix, to
+      # prevent endless loops of deploy scripts.
+      ignore:
+        - /^.*(?<!-built)$/
+    steps:
+      - checkout
+
+      # Configure build steps:
+      # - run: npm install
+      # - run: npm run build
+      #
+      # These can also be specified with a name:
+      # - run:
+      #   name: Build the thing
+      #   command: npm run build-thing
+
+      # Run the deploy:
+      - deploy:
+          command: ci/deploy.sh
+```
+
+### [Travis CI](https://travis-ci.com)
+
+1. Add the config to your repository
+2. Create and add a deploy key to your repository settings on Travis
```

Comment:
Same as with Circle, should call out the `before_script` section.

Reply:
Addressed in https://github.com/Automattic/vip-go-skeleton/pull/20/commits/559ce1fc966f3e3160ee65f9c94238401fe75295